start/kernel.ts

```diff
@@ -38,4 +38,6 @@ Server.middleware.register([() => import('@ioc:Adonis/Core/BodyParser')])
 | Route.get('dashboard', 'UserController.dashboard').middleware('auth')
 |
 */
-Server.middleware.registerNamed({})
+Server.middleware.registerNamed({
+    auth: () => import('App/Middleware/Auth')
```

Comment:
How does this name affect the rest of the coding experience?

Reply:
The name effect clearly suggests that methods should be camelCased and therefore follow that convention when dealing with creating methods etc and this auth middleware is used to guard the routes against the un-authenticated requests.